Updates steering members based on 2020 elections

bsnchan opened the pull request:
# Changes

Updates steering members based on 2020 elections in peribolos and steering-committee.md.

evankanderson commented on STEERING-COMMITTEE.md:
```diff
@@ -90,13 +90,11 @@ first name):
-|                                                                | Open \*          | Google       |                                                      |
+| <img width="30px" src="https://github.com/vaikas.png">         | Ville Aikas      | VMware       | [@vaikas](https://github.com/vaikas)                 |
 
 \* Open seats have been allocated to the organization per the prior committee
```
Do we still need this note on open seats?

bsnchan replied:
good catch, this should be fixed now :)

evankanderson reviewed (commented):
/lgtm
/hold

One comment

evankanderson commented:
Do you want to update OWNERS/OWNERS_ALIASES, too?

bsnchan commented:
@evankanderson good catch! I've updated the OWNER_ALIASES file, removed the comment around open seats and caught a a few more places that I initially missed in `peribolos/knative.yaml`

evankanderson reviewed (commented):
/lgtm

evankanderson reviewed (commented):
/hold cancel
